client-v2/src/shared/constants/theme.ts

```diff
@@ -1,23 +1,63 @@
-// @flow
+import baseStyled, { ThemedStyledInterface } from 'styled-components';
+
+const colors = {
+  blackDark: '#121212', // darkest
+  blackLight: '#333',
+  greyDark: '#515151',
+  greyMedium: '#999999',
+  greyLight: '#A9A9A9',
+  greyLightPinkish: '#C9C9C9',
+  whiteFour: '#DCDCDC',
```

Comment:
Ahh, the eternal struggle 😄I agree there are lots of whites, and I'm having a pointless struggle with myself over whether it's worth having a `lightest`, `lighter`, `light`, which might be easier for developers to use without checking the hexes. I'd be interested to hear @Reettaphant and @RichieAHB's thoughts!

Reply:
I guess the only answer here is what makes sense to a reader most. For me I think I'd prefer `light[er]` but then given we're just a command-click away maybe it's not an issue.

Reply:
I've changed the numbers in the whites to light, medium, dark.  
I was also thinking if might be useful to add a styleguide somewhere to the documentation?

 
<img width="758" alt="screen shot 2019-01-16 at 11 24 55" src="https://user-images.githubusercontent.com/32312712/51246368-3b216d00-1982-11e9-8cb7-280fb247a835.png">
<img width="1020" alt="screen shot 2019-01-16 at 11 27 19" src="https://user-images.githubusercontent.com/32312712/51246372-3ceb3080-1982-11e9-8845-b6d36dbd4bb2.png">
<img width="1275" alt="screen shot 2019-01-16 at 11 28 16" src="https://user-images.githubusercontent.com/32312712/51246374-3eb4f400-1982-11e9-8ce0-93479536c698.png">
<img width="765" alt="screen shot 2019-01-16 at 11 29 52" src="https://user-images.githubusercontent.com/32312712/51246376-3fe62100-1982-11e9-80f8-dc3db30cf704.png">